File: Python/pythonPractice/fibonacci.py
def fib(n):
	if(n == 1 or n == 2):
		return n-1
	else:
		return fib(n-1)+fib(n-2)
	
m = int(input("Enter a number:\n"))
print("The number at position no",m,"is",fib(m))

File: java/college/Fibonacci.java
package college;


import java.util.Scanner;

public class Fibonacci {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;
        System.out.print("Enter number : ");
        n = sc.nextInt();
        int a = 0, b = 1, ans = 0;

        for(int i = 0; i <= n; i++){
            System.out.print(a + " ");
            ans = a+b;
            a = b;
            b = ans;
        }
    }
}
